### ISSUE TITLE
Accessibility [1.3.1]: Labels for login/register and reset password must be explicitly associated (with "for" and "id")

### ISSUE THREAD

**Issue by yuktea** [openfoodfacts/openfoodfacts-server#6576]
### Is your feature request related to a problem? Please describe.

This issue was reported by the Accessibility Foundation (https://github.com/openfoodfacts/openfoodfacts-server/issues/4640) in the [Accessibility_report.pdf](https://github.com/openfoodfacts/openfoodfacts-server/files/5702582/Report.OpenFoodFact.15-12-2020.pdf) [read: 1.3.1]

Implicit (`<label>` Username or password `<input>` text `</input>` `</label>`) is insufficient for the screen readers to process.

### Describe the solution you'd like

Explicit labels should be implemented for all input fields
